test/replay_test.js

```diff
@@ -46,6 +48,9 @@ describe('Replay', function() {
       it('should return response trailers', function() {
         assert.deepEqual(response.trailers, { });
       });
+      it('should have matched', function() {
+        assert.equal(Replay.notMatchedFixtures.filter(matcher => matcher.path === '/weather?c=94606').length, 0);
+      })
```

Comment:
Are we missing a test for when a fixture is unmatched?

Reply:
unsure what you mean, but I noticed that it doesn't use the temp variable I created.. This I fixed now :) Also renamed to unmatched.
Let me know if that solves your question or otherwise please elaborate, thanks :)